types/index.d.ts

```diff
@@ -448,46 +454,37 @@ declare module "miragejs/orm/schema" {
      * @param data Optional initial values for model attributes/relationships
      */
     create<
-      K extends keyof Registry,
-      Init extends Registry[K],
+      K extends keyof R,
+      Init extends R[K],
       Data extends Partial<ModelInitializer<Init>>
     >(
       modelName: K,
       data?: Data
     ): Init & { [K in keyof Init & keyof Data]: Exclude<Init[K], undefined> };
 
     /** Locates one or more existing models of the given type by ID(s). */
-    find<K extends keyof Registry>(type: K, id: string): Registry[K] | null;
-    find<K extends keyof Registry>(
-      type: K,
-      ids: string[]
-    ): Collection<Registry[K]>;
+    find<K extends keyof R>(type: K, id: string): R[K] | null;
+    find<K extends keyof R>(type: K, ids: string[]): Collection<[K]>;
```

Comment:
This should be:
```suggestion
    find<K extends keyof R>(type: K, ids: string[]): Collection<R[K]>;
```
then
```ts
schema.find("foo", ["123"]).models[0];
```
in `types/tests/schema-test.ts` will return the correct type again.

Reply:
Thanks for the suggestion. Just updated. What do you think?